ISISComputingGroup/ibex_gui
# Ticket966: IBEX Usability

To Test, when editing configurations:
- Confirm that you can delete blocks, groups and PV Values using the DEL key
- Confirm that you can open the block editing dialogue by double clicking on a block
- Confirm that you can transfer selected blocks in the Groups tab by double clicking
- Confirm that you can transfer selected components by double clicking
